### ISSUE TITLE
Keyboard navigation doesn't work in the ASM tree

### ISSUE THREAD

**Issue by niklas-ourmachinery** [OurMachinery/themachinery-public#287]
Left/Right/Up/Down/Space should work to navigate the ASM tree, just as they do in the entity tree.
